pkg/chartutil/values_test.go

```diff
@@ -316,6 +322,8 @@ func TestCoalesceValues(t *testing.T) {
 		expect string
 	}{
 		{"{{.top}}", "yup"},
+		// The keys bottom, right, left, front, and back should not be present.
+		{"{{.side}}", ""},
```

Comment:
These look great, we do need to test that all of the other keys are unset. This was a learning experience for me too, but I found out it should look like this: `{"{{.bottom}}", "<no value>"},`